check_binary.sh

```diff
@@ -27,7 +27,7 @@ if [[ "$PACKAGE_TYPE" == libtorch ]]; then
   # NOTE: Only $PWD works on both CentOS and Ubuntu
   install_root="$PWD"
 else
-  py_dot="${DESIRED_PYTHON:0:3}"
+  py_dot="${DESIRED_PYTHON:0:4}"
```

Comment:
why don't we just use DESIRED_PYTHON instead of py_dot then if the variable is unchanged?

Reply:
I think we want to make sure only required number of digits get in - no minor verisons:)

Reply:
Would it be better to just do a `cut` based on using `.` as our delimeter?

Or even just set `py_dot` to `DESIRED_PYTHON` if it accurately reflects what we expect.

i.e. if `DESIRED_PYTHON` matches `[1-9]*\.[1-9]+` then we just set `py_dot=${DESIRED_PYTHON}`